src/launch/automation/provider/az/functions.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import re
+import subprocess
+
+logger = logging.getLogger(__name__)
+
+
+def callback_deploy_remote_state(
+    key,
+    value,
```

Comment:
These annotations seem right from the below code, but if not, feel free to correct.

```suggestion
    key: str,
    value: str,
```